### PR TITLE
Clean-up README

### PR DESCRIPTION
**Pull request by kdamaszk**
# What does this PR do?

Cleans up README. Add more information how to setup TGI

### CODE REVIEW

**Comment by kdamaszk** [comment]
@regisss @libinta please take a look on this. I reorganized README a little bit. Let me know what do you think.

File: README.md [hunk 1]
```diff
@@ -29,71 +29,77 @@ To use [🤗 text-generation-inference](https://github.com/huggingface/text-gene
...
    ```bash
    curl 127.0.0.1:8080/generate \
      -X POST \
-     -d '{"inputs":"What is Deep Learning?","parameters":{"max_new_tokens":17, "do_sample": true}}' \
+     -d '{"inputs":"What is Deep Learning?","parameters":{"max_new_tokens":32}}' \
      -H 'Content-Type: application/json'
    ```
-   > The first call will be slower as the model is compiled.
-5. To run benchmark test, please refer [TGI's benchmark tool](https://github.com/huggingface/text-generation-inference/tree/main/benchmark).
+5. To run static benchmark test, please refer [TGI's benchmark tool](https://github.com/huggingface/text-generation-inference/tree/main/benchmark).
```
Comment (regisss) [inline README.md]:
```suggestion
5. To run static benchmark test, please refer to [TGI's benchmark tool](https://github.com/huggingface/text-generation-inference/tree/main/benchmark).
```

File: README.md [hunk 2]
```diff
@@ -29,71 +29,77 @@ To use [🤗 text-generation-inference](https://github.com/huggingface/text-gene
...
    * `docker exec -it <docker name> bash` , pick the docker started from step 3 or 4 using docker ps
    * `text-generation-benchmark -t <model-id>` , pass the model-id from docker run command
    * after the completion of tests, hit ctrl+c to see the performance data summary.
 
-> For gated models such as [StarCoder](https://huggingface.co/bigcode/starcoder), you will have to pass `-e HUGGING_FACE_HUB_TOKEN=<token>` to the `docker run` command above with a valid Hugging Face Hub read token.
-
 For more information and documentation about Text Generation Inference, checkout [the README](https://github.com/huggingface/text-generation-inference#text-generation-inference) of the original repo.
 
 Not all features of TGI are currently supported as this is still a work in progress.
+TGI on Intel Gaudi has been validated mainly with Llama model. Support for other models from Optimum Habana will be added successively
```
Comment (regisss) [inline README.md]:
```suggestion
TGI on Intel Gaudi has been validated mainly with Llama model. Support for other models from Optimum Habana will be added successively.
```

**Review by regisss — COMMENTED** [review]
Nice! I just left a couple of nits and after that we can merge!

**Review by regisss — APPROVED** [review]

**Pull request merged**